{{cookiecutter.project_folder}}/pyproject.toml

```diff
@@ -0,0 +1,82 @@
+[tool.poetry]
+name =  "{{ cookiecutter.namespace + '-' + cookiecutter.module_name if cookiecutter.namespace else cookiecutter.module_name }}"
+version = "{{ cookiecutter.version }}"
+description = "{{ cookiecutter.short_description }}"
+authors = ["{{ cookiecutter.full_name }} <{{ cookiecutter.email }}>"]
+readme = "README.md"
+packages = [{include =  "{{ cookiecutter.namespace + '/' + cookiecutter.module_name if cookiecutter.namespace else cookiecutter.module_name }}"}]
+
+[tool.poetry.dependencies]
+python = ">={{ cookiecutter.python_version }}, <3.12"
+{%- if cookiecutter.use_jupyterlab == "y" %}
+jupyterlab = "^3.0.0"
+{%- endif %}
+{%- if "{{ cookiecutter.command_line_interface|lower }}" == "cli" %}
+typer-cli = "^0.0.13"
+{%- endif %}
+{%- if cookiecutter.use_mypy == "y" %}
+pydantic = "^1.10.7"
+{%- endif %}
+
+[tool.poetry.group.dev.dependencies]
+black = {version = "^23.1.0", allow-prereleases = true}
+{%- if cookiecutter.use_isort == "y" %}
+isort = "^5.12.0"
+{%- endif %}
+{%- if cookiecutter.use_ruff == "y" %}
+ruff = "^0.4.4"
+{%- endif %}
+{%- if cookiecutter.use_flake8 == "y" %}
+flake8 = "^6.0.0"
+pyproject-flake8 = "^6.0.0"
+{%- endif %}
+{%- if cookiecutter.use_mypy == "y" %}
+mypy = "^0.991"
+{%- endif %}
+{%- if cookiecutter.use_bandit == "y" %}
+bandit = "^1.7.4"
+{%- endif %}
+{%- if cookiecutter.use_pylint == "y" %}
+pylint = "^2.16.1"
+astroid = "^2.14.1"
+{%- endif %}
+{%- if cookiecutter.use_precommit == "y" %}
```

Comment:
Precommit is mandatory, so let's remove this option

Reply:
Removed conditional